sdlib/d/gc/bin.d

```diff
@@ -30,20 +30,31 @@ struct Bin {
 		import d.gc.slab;
 		auto size = binInfos[sizeClass].itemSize;
 
+		auto needFill = buffer.length;
```

Comment:
If you plan on filling the buffer, then the best is likely to slice the remaining part of the buffer at each iterration and check that the size of the buffer is greater than 0 int he loop.

But I'm wondering if we are not better off having a minimal number of allocation requested as a parameter, and possibly a bigger buffer, so that we can fill in on a slab granularity.

Reply:
The latter strategy is done by other allocators, but not jemalloc, I'm not sure why.

Reply:
512 items for small size classes do not seem absurd.